java/src/jmri/jmrix/tmcc/SerialThrottle.java

```diff
@@ -65,7 +65,8 @@ public void setFunction(int func, boolean value) {
         0x0010, 0x0011, 0x0012, 0x0013, 0x0014, /** 5-9 */
         0x0015, 0x0016, 0x0017, 0x0018, 0x0019, /** 10-14 */
         0x0009, 0x001E, 0x0000, 0x0003, 0x0001, /** 15-19 */
-        0x0004, 0x0007 /** 20-21 */
+        0x0004, 0x0007, 0x0047, 0x0043, 0x0028, /** 20-24 */
+        0x0029, 0x002A, 0x002B, /** 25-27 */    
```

Comment:
I think that the trailing comma needs to be suppressed here.

Reply:
Thanks for checking this PR.

Java has no problem with the comma. I checked it now on my local computer and it handles SERIAL_FUNCTION_CODES correct. And all CI has passed.

But if you want it changed anyway then I can ask the orginal author to change it. The PR is from his branch so I cannot do the change myself.

Reply:
Happy to have this merged as-is - we can follow this up later.